src/NRedisStack/Json/JsonCommands.cs

```diff
@@ -60,6 +60,37 @@ public bool Set(RedisKey key, RedisValue path, RedisValue json, When when = When
         return true;
     }
 
+    /// <inheritdoc/>
+    public bool SetFromFile(RedisKey key, RedisValue path, string filePath, When when = When.Always)
+    {
+        string fileContent  = File.ReadAllText(filePath);
+        // TODO: Check if the content is valid JSON
```

Comment:
There is no need to validate that on the client side, RedisJSON will validate that for you

Reply:
If I do the validation check in the code before I send the json to Redis, it will prevent unnecessary calls to Redis and improve runtime in cases where the file is not correct. What do you think?

Reply:
You'll improve the runtime of errors by making the "working path" slower..

Reply:
@chayim

Reply:
I'm generally a fan of doing defensive things in a client. The cost of this is fairly minimal - under the covers it's a single call to STAT on the os (not even an open) .

Running through this 10M times locally, including the check took under 0.3s.  IMHO that's worth it - and it gives the user the ability to catch the exception if they care.

Reply:
I'm advocating for validating the file exists only **not it's contents**. I feel like that latter is reasonable for the user.